core/client/src/main/java/alluxio/client/file/options/CompleteUfsFileOptions.java

```diff
@@ -98,10 +98,10 @@ public String toString() {
    */
   public CompleteUfsFileTOptions toThrift() {
     CompleteUfsFileTOptions options = new CompleteUfsFileTOptions();
-    if (!mPermission.getOwner().isEmpty()) {
+    if (mPermission.getOwner() != null && !mPermission.getOwner().isEmpty()) {
```

Comment:
Do `null` and `` imply different things for this field? Otherwise we could try to just consolidate it to `` and prevent the value from being null.